src/test/java/SetTest.java

```diff
@@ -0,0 +1,49 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author jeongheekim
+ * @date 3/6/24
+ */
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("numbers의 size는 3이다.")
+    void setSizeTest() {
+        assertEquals(numbers.size(), 3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

assertj api 엔 이런것도 있어요!

Reply:
조금 더 가독성 있는 코드가 되겠군요